domains/certificates/Query.API/API/GranularCertificateIssuer/EnergyMeasuredConsumer.cs

```diff
@@ -60,13 +60,14 @@ public async Task Consume(ConsumeContext<EnergyMeasuredIntegrationEvent> context
 
     private static bool ShouldEventBeProduced(MasterData? masterData)
     {
+        var now = DateTime.Now;
```

Comment:
```suggestion
        var now = DateTimeOffset.UtcNow;
```

Technically not needed in this case, but as a default I would go with using DateTimeOffset over DateTime as I am then always sure to have a single point in time.